### PR TITLE
POJOs were refactored

### PR DESCRIPTION
**Pull request by rulsan**

### CODE REVIEW

File: src/test/java/pojo/Film.java [hunk 1]
```diff
@@ -11,6 +13,16 @@ public class Film {
     private List<String> charactersHTTP;
     private String url;
 
+    public Film() {}
+
+    @JsonCreator
+    public Film(@JsonProperty("title") String title, @JsonProperty("characters") List<String> charactersHTTP, @JsonProperty("url") String url) {
```
Comment (maksymYena) [inline src/test/java/pojo/Film.java]: Could u please "beautify" the constructor arguments (not in the one line)? You can use the sonar or linter

File: src/test/java/pojo/Pilot.java [hunk 2]
```diff
@@ -10,6 +12,16 @@ public class Pilot {
     private List<String> starshipsHTTP;
     private String url;
 
+    public Pilot() {}
+
+    @JsonCreator
+    public Pilot(@JsonProperty("name") String name, @JsonProperty("starships") List<String> starshipsHTTP, @JsonProperty("url") String url) {
```
Comment (maksymYena) [inline src/test/java/pojo/Pilot.java]: The same here

File: src/test/java/pojo/Starship.java [hunk 3]
```diff
@@ -35,11 +52,37 @@ public void setPilots(List<String> pilots) {
...
         return url;
     }
 
     public void setUrl(String url) {
         this.url = url;
     }
+
+    @Override
+    public String toString() {
+        return "The Starship is " + name + '\'' + ", starship_class='" + starship_class + '\'' + ", pilotsHTTP=" + this.getPilots().toString() + ", url='" + url + '\'';
```
Comment (maksymYena) [inline src/test/java/pojo/Starship.java]: The same here

File: src/test/java/pojo/Starship.java [hunk 4]
```diff
@@ -35,11 +52,37 @@ public void setPilots(List<String> pilots) {
...
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Starship starship = (Starship) o;
+        return name.equals(starship.name) && starship_class.equals(starship.starship_class) && pilotsHTTP.equals(starship.pilotsHTTP) && url.equals(starship.url);
```
Comment (maksymYena) [inline src/test/java/pojo/Starship.java]: The same here

File: src/test/java/pojo/Starship.java [hunk 5]
```diff
@@ -35,11 +52,37 @@ public void setPilots(List<String> pilots) {
...
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+
+        result = prime * result + ((this.getName() == null ? 0 : this.getName().hashCode()));
+        result = prime * result + ((this.getStarship_class() == null ? 0 : this.getStarship_class().hashCode()));
+        result = prime * result + ((this.getPilots().isEmpty() ? 0 : this.getPilots().hashCode()));
+        result = prime * result + ((this.getUrl() == null ? 0 : this.getUrl().hashCode()));
+        return result;
```
Comment (maksymYena) [inline src/test/java/pojo/Starship.java:87]: No other objections

**Review by maksymYena — CHANGES_REQUESTED** [review]

File: src/test/java/pojo/Film.java (hunk 1 above)
Comment (rulsan) [reply src/test/java/pojo/Film.java]: The code was modified

File: src/test/java/pojo/Pilot.java (hunk 2 above)
Comment (rulsan) [reply src/test/java/pojo/Pilot.java]: The code was modified

File: src/test/java/pojo/Starship.java (hunk 3 above)
Comment (rulsan) [reply src/test/java/pojo/Starship.java]: The code was modified

File: src/test/java/pojo/Starship.java (hunk 4 above)
Comment (rulsan) [reply src/test/java/pojo/Starship.java]: The code was modified

**Review by maksymYena — APPROVED** [review]
LGTM

**Pull request merged**